utils_test.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // Reset random reader to original value
 func resetRandReader() {
-	RandReader = rand.Reader
+	randReader = rand.Reader
```

Comment:
Let's keep this one public. It was exposed on purpose to make it possible to test against deterministic test vectors in client libraries (which I think is a reasonable request/thing to do).